cmd/image-builder/config.go

```diff
@@ -4,12 +4,14 @@ import (
 	"fmt"
 	"os"
 
+	adoPipelines "github.com/kyma-project/test-infra/pkg/azuredevops/pipelines"
 	"github.com/kyma-project/test-infra/pkg/sign"
 	"github.com/kyma-project/test-infra/pkg/tags"
 	"gopkg.in/yaml.v3"
 )
 
 type Config struct {
+	adoPipelines.Config `yaml:",inline" json:",inline"`
```

Comment:
IMO this should be under subconfiguration, as all other configurations to keep consistency in the code.
```suggestion
	AdoConfig adoPipelines.Config `yaml:"ado-config,omitempty" json:"ado-config,omitempty"`
```